packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -805,53 +822,71 @@ function UpgradeCreditInformation( { product }: { product: ResponseCartProduct }
 	if ( isMonthlyProduct( product ) ) {
 		return (
 			<>
-				{ translate( 'Upgrade Credit: %(upgradeCredit)s applied in first month only', {
-					comment:
-						'The upgrade credit is a pro rated balance of the previous plan which is to be applied' +
-						'as a deduction to the first year of next purchased plan. It will be applied once only in the first term',
-					args: {
-						upgradeCredit: formatCurrency( upgradeCredit, product.currency, {
-							isSmallestUnit: true,
-							stripZeros: true,
-						} ),
-					},
-				} ) }
+				{ translate(
+					'Upgrade Credit: %(upgradeCredit)s applied in first month only {{UpgradeCreditHelpIconLink}}{{/UpgradeCreditHelpIconLink}}',
```

Comment:
Is there any way to add this icon to the rendered output without changing the translated string? Could you just add `<UpgradeCreditHelpIconLink />` immediately after the translated string inside the `return` statement?

The reason I ask is that changing the string will require a retranslation and is limiting in case we ever want to adjust the icon position.

Reply:
Good catch! I have made this change.

Thanks for the review, @sirbrillig! Unfortunately, I had to move this PR to a draft stage since the spec for this task has [changed](https://github.com/Automattic/growth-foundations/issues/143#issuecomment-1661226433).